mne/io/fieldtrip/utils.py

```diff
@@ -203,12 +203,18 @@ def _set_sfreq(ft_struct):
         sfreq = ft_struct['fsample']
     except KeyError:
         try:
-            t1 = ft_struct['time'][0]
-            t2 = ft_struct['time'][1]
-            difference = abs(t1 - t2)
-            sfreq = 1 / difference
+            time = ft_struct['time']
         except KeyError:
             raise ValueError('No Source for sfreq found')
+        else:
+            t1, t2 = float(time[0]), float(time[1])
+            sfreq = 1 / abs(t1 - t2)
```

Comment:
Why not simply
```suggestion
            sfreq = 1 / (t2 - t1)
```
?

Reply:
Wanted to change the underlying code minimally, wasn't sure if there was a reason. But I agree `t2 - t1` is probably fine